src/app/hooks/finalityProviders/useMobileSortModal.ts

```diff
@@ -0,0 +1,22 @@
+import { useState } from "react";
+
+import { SortField } from "@/app/types/finalityProviders";
+
+export const useMobileSortModal = () => {
+  const [isSortModalOpen, setIsSortModalOpen] = useState(false);
+  const [activeOrder, setActiveOrder] = useState(0);
+  const [selectedCriterion, setSelectedCriterion] =
+    useState<SortField>("stakeSat");
```

Comment:
Same concern with `stakeSat` not being extracted as a constant, even though TS helps with that I would prefer using a constant that holds the "stakeSat" value